README.md

```diff
@@ -3,8 +3,8 @@
 EHRbase is an [openEHR](openehr.org) Clinical Data Repository, providing a standard-based backend for interoperable clinical applications. It implements the latest version of the openEHR Reference Model (RM 1.0.4) and version 1.4 of the Archetype Definition Language (ADL). Applications can use the capabilities of EHRbase through the latest version of the [openEHR REST API](https://specifications.openehr.org/releases/ITS-REST/latest/) and model-based queries using the [Archetype Query Language](https://specifications.openehr.org/releases/QUERY/latest/AQL.html).
 
 ## Release Notes
-##### 2019, October 10
-This initial release is an alpha version. This means that most endpoints of the official openEHR REST API are implemented. DIRECTORY and EHR_STATUS will become available within a few couple of weeks. There are known issues and we are heavily working to get to beta status. Please see [EHRbase Documentation](https://docs.ehrbase.org/en/latest/) for further details. For now, we encourage the use of EHRbase for the purpose of development of openEHR applications and to make trial implementations of changes applicable to the openEHR Specification. Deployment in production with real patient data is not recommended yet.
+##### 2020, January 10
+This release brings EHRbase from a pre-alpha to an alpha version. Besides numerous bug fixes, EHRbase provides new endpoints based on the openEHR REST API to handle FOLDERS and the EHR_STATUS objects. This version also introduces Spring Transactions, improved handling of the system ID and AQL queries with partial paths finally return data in canonical json format (including full compositions). The conformance tests have been enhanced by adding AQL tests. Please see [EHRbase Documentation](https://docs.ehrbase.org/en/latest/) for further details. We still encourage the use of EHRbase for the purpose of development of openEHR applications and to make trial implementations of changes applicable to the openEHR Specification. However, a deployment in production with real patient data is not recommended yet.
```

Comment:
I think a direct link to the release notes in the docs would be great. We can publish the new docs version before we push the changes here, so that would work workflow-wise.